app/src/main/java/com/yeonae/chamelezone/view/course/adapter/CourseTabRvAdapter.kt

```diff
@@ -4,17 +4,17 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.yeonae.chamelezone.R
-import com.yeonae.chamelezone.data.model.Course
 import com.yeonae.chamelezone.ext.glideImageSet
+import com.yeonae.chamelezone.network.model.CourseResponse
 import kotlinx.android.synthetic.main.item_course_list.view.*
 
-class CourseTabRvAdapter(private val courseList: ArrayList<Course>) :
+class CourseTabRvAdapter() :
     RecyclerView.Adapter<CourseTabRvAdapter.CourseViewHolder>() {
-
+    private var items = mutableListOf<CourseResponse>()
```

Comment:
```suggestion
    private val items = mutableListOf<CourseItem>()
```
response를 그대로 view의 item으로 사용하지 말고 변환된 class를 사용해주세요.
또, var 타입일 필요가 없을 것 같습니다.

Reply:
수정했습니당